### PR TITLE
Update mappings, include .tar.gz for easier use and self-document build process 

### PR DESCRIPTION
**Pull request by gnarvaja**
- New version 0.0.6 generated with latest unidecode
- Change in builder.py to use repository in arbitrary path and self-document that you have to clone the python repository.
- Switch from zip to tar.gz since unzip is not available in postgres docker images, and tar / gzip are usually available (at least in Linux).
- Remove versioned sql from source, now are generated in the dist step.
- Include generated .tar.gz in the sources to be able to download it from github.

### CODE REVIEW

File: META.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "name": "unidecode",
   "abstract": "ASCII transliterations of Unicode text",
-  "version": "0.0.4",
-  "maintainer": "Alexander Kuznetsov <alexkuz@gmail.com>",
+  "version": "0.0.6",
+  "maintainer": "Guillermo Narvaja <guillermo.narvaja@radiocut.fm>",
```
Comment (alexkuz) [inline META.json]: Can you make it a list instead? So there'd be both of us.

Comment (gnarvaja) [reply META.json]: Done. Sorry I changed the maintainer before considering sending the PR, now we are both as maintainers.

**Comment by alexkuz** [comment]
Is the README still relevant, BTW? I didn't follow all that for a really long time.

Also, wouldn't it be better to publish .tar.gz as a release asset rather than storing it in repo?

**Comment by gnarvaja** [comment]
Yes, perhaps adding the .tar.gz as release asset it's better. I didn't
consider that alternative. The release should be created in your repo, I
think.

About the README, I think it's fine, at least the "make && make install"
works fine, and the first section too.

--

Guillermo M. Narvaja

RadioCut - https://radiocut.fm

Phone: 54-911-6783-4435 (WhatsApp)

Telegram/WeChat: gnarvaja

Email: guillermo.narvaja@radiocut.fm

Iguazú 656 - Ciudad de Buenos Aires - Argentina

**Comment by gnarvaja** [comment]
I just removed the .tar.gz from the PR and added as release asset.

--

Guillermo M. Narvaja

RadioCut - https://radiocut.fm

Phone: 54-911-6783-4435 (WhatsApp)

Telegram/WeChat: gnarvaja

Email: guillermo.narvaja@radiocut.fm

Iguazú 656 - Ciudad de Buenos Aires - Argentina

**Comment by alexkuz** [comment]
I've added you as a collaborator, feel free to create a release here

**Pull request merged**